buildpack.md

```diff
@@ -209,21 +213,19 @@ If a cache is provided the lifecycle:
 - SHOULD store the Layer Content Metadata associated with each layer so that it can be recovered using the layer Diff ID
 
 Before the next re-build:
-- The lifecycle:
-  - MUST either restore the entire `<layers>/<layer>.toml` file and corresponding `<layers>/<layer>/` directory from any previous build to the same paths or
-  - MUST restore neither the `<layers>/<layer>.toml` file nor corresponding `<layers>/<layer>/` directory.
+- The lifecycle MUST do both or neither of the following:
+  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `launch`, `build`, and `cache` booleans.
+  - Restore layer contents to the `<layers>/<layer>/` directory.
+
+#### Ignored Layers
 
-#### Other Layers
+Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
 
-The lifecycle:
-- If `build = false`
-  - MUST NOT make the layer available to subesequent buildpacks.
-- If `launch = false`
-  - MUST NOT include the layer or the Layer Content Metadata in the OCI image.
-- If `cache = false`
-  - MUST NOT store the layer or the Layer Content Metadata in the cache.
+At the end of each individual buildpack's build phase:
+- The lifecycle:
+  - MUST rename `<layers>/<layer>/` to `<layers>/<layer>.ignore/` for all layers where `launch = false`, `build = false`, and `cache = false`, in order to prevent subsequent buildpacks from accidentally depending on an ignored layer.
 
-Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, that exist for the duration of a single build.
+The lifecycle restores all layers from previous builds as ignored layers. Buildpacks MUST set `launch`, `build`, and/or `cache` in `<layers>/<layer>.toml` to use a restored layer.
```

Comment:
It might be worth clarifying that "as ignored layers" means the types flags are set to false in the toml file, otherwise this could be interpreted to mean that the directory is also named with .ignore.